AGXUnity/Utils/Plot.cs

```diff
@@ -0,0 +1,196 @@
+using UnityEngine;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace AGXUnity.Utils
+{
+  [DisallowMultipleComponent]
+  public class Plot : ScriptComponent
+  {
+    /// <summary>
+    /// Native AGXPlot.System object.
+    /// </summary>
+    public agxPlot.System Native { get { return m_plotSystem; } }
```

Comment:
Similar to my other comment I think we should use implicit backing field where possible and short form for properties

Reply:
Fixed.